packages/webapp/src/routes/land/client-individual-organisation.js

```diff
@@ -0,0 +1,36 @@
+import constants from '../../utils/constants.js'
+
+const handlers = {
+  get: async (request, h) => {
+    return h.view(constants.views.CLIENT_INDIVIDUAL_ORGANISATION)

```

Comment:
Can the Get pre-populate to assist with CYA routing?

Reply:
done